module/VuFind/tests/unit-tests/src/VuFindTest/Log/LoggerTest.php

```diff
@@ -51,7 +53,9 @@ class LoggerTest extends \PHPUnit\Framework\TestCase
      */
     public function testLogException()
     {
-        $callback = function ($a): bool {
+        $callback = function ($level, $message, $context = []) use (&$capturedArgs): bool {
+            $capturedArgs = func_get_args();
```

Comment:
What is the purpose of `$capturedArgs`? It doesn't seem to be getting used for anything; apologies if I'm missing or misunderstanding something!

Reply:
I guess it should be removed then -- looks like it's still here.